www/viewgame

```diff
@@ -1719,7 +1951,81 @@ if ($ratingCntTot == 0 && $memberReviewCnt == 0) {
             . "$ratingCntAvg rating" . ($ratingCntAvg != 1 ? "s" : "")
             . "</a>)<br>";
     }
+// &&&&&&&&&&&&&&&&&&&&&&&&&&& Move the new times stuff here
+
+//******find median function starts here
+function findMedian($array_input) {
+       if(is_array($array_input)) {
+           // Sort the entries in the array
+           sort($array_input);
+           // Divide the number of entries by two
+           $total_entries = count($array_input);
+           $half = $total_entries / 2;
+           $int_half = (int) $half;      // Find just the integer part of a number, in case it's not a whole number
+    
+           // Calculate median 
+           if ($total_entries % 2 != 0) {            // Dividing by 2 gives a remainder, so the array must have an odd number of entries
+            $median = $array_input[$int_half];	     // Choose the middle entry
+
+           } else {                                  // The array has an even number of entries, so there will be two middle entries
+           $middle1 = $array_input[$int_half-1];     // Find the first middle entry
+           $middle2 = $array_input[$int_half];      // Find the second middle entry
+           $median = ($middle1 + $middle2) / 2;
+           }
+       return $median;
+       }    
+   }
+
+// **** Find Median Function Ends Here
+
+//***** Query for Times sectionbegins here */
+$result = mysqli_execute_query($db, "select timevote from playertimes where gameid = ?", [$id]);
+if (!$result) throw new Exception("Error: " . mysqli_error($db));
+$alltimes= array();
+while ($row = mysqli_fetch_assoc($result)) {
+    $time=$row['timevote'];
+    $alltimes[]=$time;
```

Comment:
If `$time` is ununsed, this can be single line: `$alltimes[] = $row['timevote'];`